cypress/support/cypress-support/src/main.js

```diff
@@ -189,6 +204,13 @@ export default function (module) {
             }
           });
         }
+        if (UTILS.getEnvVariable(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED, false) == true) {
+          cy.startOrStopInteractionsService(CONSTANTS.STOPPED).then((response) => {
+            if (response) {
+              Cypress.env(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED, false);
+            }
+          });
+        }
```

Comment:
I am not sure I understand what line 207 means . and what is 210 trying to do

Reply:
here we are stopping the service if this enabled in before block